doc/player_setup.md

```diff
@@ -222,20 +222,58 @@ source = process:///usr/bin/mpv?name=Webradio&sampleformat=48000:16:2&params=htt
 
 ### Line-in
 
+#### ALSA Method
+Audio can be played directly through the line-in via ALSA. The following guide was written in regards to a raspberry pi using a HiFiBerry product but it should roughly apply for line-in with other hardware.
+
+1. Get a list of recording devices from ALSA.
+    ```
+    arecord -l
+    ```
+2. You will receive output in the following format. Note the `<device_name>` of your input device for the next step.
+    ```
+    card <number>: <device_name> ....
+    ```
+3. Edit the file `/etc/snapserver.conf` and add the following line, substituting `<device_name>` for the value derived from the previous step. Pick whatever you'd like for `<stream_name>`.
+    ```
+    stream = alsa://?name=<stream_name>&device=hw:<device_name>
```

Comment:
I tried to use the card number instead of its name. Works flawlessly

Reply:
> I tried to use the card number instead of its name. Works flawlessly

So just be careful using the numeric value, I had the number change on me on reboot.